ci/generate_docs.sc

```diff
@@ -29,7 +29,10 @@ val currentGitBranch = %%('git, "rev-parse", "--abbrev-ref", "HEAD")(pwd).out.li
 var overrideTargetVersions = Map.empty[String, GitCheckoutable]
 
 @main
-def main(release_commits_override: Map[String, String] = Map.empty) = {
+def main(release_commits_override: Map[String, String] = Map.empty,
```

Comment:
As I mentioned before, I'd define multiple targets as done in `ci/pipeline`. This way one can test each step.